README.md

```diff
@@ -1793,6 +1794,77 @@ async function getCleanCodeArticle() {
 **[⬆ back to top](#table-of-contents)**
 
 
+## **Error Handling**
+Thrown errors are a good thing! They mean the runtime has successfully
+identified when something in your program has gone wrong and it's letting
+you know by stopping function execution on the current stack, killing the
+process (in Node), and notifying you in console with a stack trace.
+
+### Don't ignore caught errors
+Doing nothing with an caught error doesn't give you the ability to ever fix
```

Comment:
Weird, I thought I had made the 'an' to 'a' grammar fix before pushing.

The 'fix' to 'fix it' isn't right though; what I have I believe is correct or else it would effectively read 'fix it said error'.